ml/src/test/scala/ai/h2o/sparkling/ml/models/H2OMOJOPipelineModelTestSuite.scala

```diff
@@ -314,4 +291,27 @@ class H2OMOJOPipelineModelTestSuite extends FunSuite with SparkTestContext with
     val settings = H2OMOJOSettings()
     testTransformAndTransformSchemaAreAligned(settings)
   }
+
+  test("Mojo pipeline can expose contribution (SHAP) values") {
+
+    val mojoSettings = H2OMOJOSettings(withContributions = true)
+    val pipeline = H2OMOJOPipelineModel.createFromMojo(
+      this.getClass.getClassLoader.getResourceAsStream("daiMojoShapley/pipeline.mojo"),
+      "pipeline.mojo",
+      mojoSettings)
+
+    val df = spark.read.option("header", "true").csv("ml/src/test/resources/daiMojoShapley/example.csv")
+    val transformed = pipeline.transform(df)
+    val predictionsAndContributions = transformed.select("prediction.*")
+
+    val featureColumns = 1
```

Comment:
that looks very clean 👍 the only possible improvements I see could be adding "expected" prefix and changing the assertion style to a "modern" one, for example:
`predictionsAndContributions.columns should have size predictionColumns + contributionColumns`

Reply:
Assertion style changed.